addons/pos_razorpay/static/src/app/payment_razorpay.js

```diff
@@ -176,6 +178,13 @@ export class PaymentRazorpay extends PaymentInterface {
         return new Promise(razorpayFetchPaymentStatus);
     }
 
+    _getPaymentDate(timeMillis) {
+        const utcDate = timeMillis
+            ? DateTime.fromMillis(timeMillis, { zone: "utc" })
```

Comment:
@paan-odoo Maybe this should be:

```suggestion
            ? DateTime.fromMillis(timeMillis)
```